editor/inspector/utils/prop.js

```diff
@@ -30,25 +30,57 @@ exports.sortProp = function(propMap) {
 
 /**
  *
- * This function can filter the contents of the dump and is mainly used to get the user's dump data
+ * This method is used to update the custom node
+ * @param {HTMLElement} container
  * @param {string[]} excludeList
  * @param {object} dump
- * @param {(element,prop)=>void} onElementCreated
+ * @param {(element,prop)=>void} update
  */
-exports.getCustomPropElements = function(excludeList, dump, onElementCreated) {
-    const customPropElements = [];
+exports.updateCustomPropElements = function(container, excludeList, dump, update) {
     const sortedProp = exports.sortProp(dump.value);
+    container.$ = container.$ || {};
+    /**
+     * @type {Array<HTMLElement>}
+     */
+    const children = [];
     sortedProp.forEach((prop) => {
         if (!excludeList.includes(prop.key)) {
-            const node = document.createElement('ui-prop');
-            node.setAttribute('type', 'dump');
-            if (typeof onElementCreated === 'function') {
-                onElementCreated(node, prop);
+            if (!prop.dump.visible) {
+                return;
+            }
+            let node = container.$[prop.key];
+            if (!node) {
+                node = document.createElement('ui-prop');
+                node.setAttribute('type', 'dump');
+                prop.dump.displayOrder = prop.dump.displayOrder === undefined ? 0 : Number(prop.dump.displayOrder);
```

Comment:
set default displayOrder value should deal in ‘sortProp’ method

Reply:
done